src/Microsoft.ApiDesignGuidelines.Analyzers/Core/IdentifiersShouldNotMatchKeywords.cs

```diff
@@ -1,69 +1,621 @@
-                                                                             customTags: WellKnownDiagnosticTags.Telemetry);
+                                                                             helpLinkUri: HelpLinkUri,
+                                                                             customTags: CustomTags);
         internal static DiagnosticDescriptor NamespaceRule = new DiagnosticDescriptor(RuleId,
                                                                              s_localizableTitle,
                                                                              s_localizableMessageNamespace,
-                                                                             DiagnosticCategory.Naming,
-                                                                             DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
+                                                                             Category,
+                                                                             Severity,
+                                                                             isEnabledByDefault: IsEnabledByDefault,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
-                                                                             customTags: WellKnownDiagnosticTags.Telemetry);
+                                                                             helpLinkUri: HelpLinkUri,
+                                                                             customTags: CustomTags);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(MemberParameterRule, MemberRule, TypeRule, NamespaceRule);
 
         public override void Initialize(AnalysisContext analysisContext)
         {
-            
+            analysisContext.RegisterSymbolAction(AnalyzeMemberRule,
+                SymbolKind.Event, SymbolKind.Method, SymbolKind.Property);
+
+            analysisContext.RegisterSymbolAction(AnalyzeMemberParameterRule,
+                SymbolKind.Method);
+        }
+
+        private void AnalyzeMemberRule(SymbolAnalysisContext context)
+        {
+            ISymbol symbol = context.Symbol;
+            if (symbol.GetResultantVisibility() != SymbolVisibility.Public)
+            {
+                return;
+            }
+
+            string matchingKeyword;
+            if (!IsKeyword(symbol.Name, out matchingKeyword))
+            {
+                return;
+            }
+
+            // IsAbstract returns true for both abstract class members and interface members.
+            if (symbol.IsVirtual || symbol.IsAbstract)
```

Comment:
I believe FxCop was checking for overridable members only and 'sealed override' would sneak through here